run.py

```diff
@@ -0,0 +1,13 @@
+import asyncio
+import os
+
+from game.grpc_interface import Interface
```

Comment:
"""
import os

import asyncio

from game.grpc_interface import Interface
"""

Imports must be done in alphabetical order and separated according to:

1. Native packages
2. Install packages
3. Project modules

Reply:
modified